models/src_amazon_ads.yml

```diff
@@ -116,11 +116,11 @@ sources:
             description: The type of targeting used for the campaign, either manual or auto.
 
       - name: campaign_level_report
-        description: Campaign reports contain performance data broken down at the campaign level. 
+        description: Campaign reports contain performance data broken down at the campaign level.  
```

Comment:
Same comment for the report table above.

Reply:
Done.